lsstsal/scripts/gensimplesample.tcl

```diff
@@ -548,12 +548,12 @@ global SAL_WORK_DIR OPTIONS
      puts $fout "	struct ackcmd \{"
      add_private_idl $fout
      puts $fout "	  long	salIndex;
-	  long	ack;
-	  long	error;
+	  long		ack;
+	  long		error;
 	  string	result;
 	  string	identity;
-	  long	origin;
-	  long	cmdtype;
+	  long		origin;
```

Comment:
See above, those doesn't seem as necessary

Reply:
These formatting changes look fine in the context of the whole file. I'd say leave them.